content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -7,6 +7,7 @@ min-kubernetes-server-version: v1.18
 <!-- overview -->
 
 {{< feature-state state="alpha"  for_k8s_version="v1.18" >}}
```

Comment:
is it common practice to leave the alpha section in? I couldn't find anything similar from a brief search

I'd imagine the older versions of the docs will have the alpha tag

Reply:
Yep, remove this line.

Reply:
done